src/deepsparse/transformers/helpers.py

```diff
@@ -30,11 +30,7 @@
 from onnx import ModelProto
 
 from deepsparse.log import get_main_logger
-from deepsparse.utils.onnx import (
-    _MODEL_DIR_ONNX_NAME,
-    model_to_path,
-    truncate_onnx_model,
-)
+from deepsparse.utils.onnx import MODEL_ONNX_NAME, model_to_path, truncate_onnx_model
```

Comment:
can we double check the changes to `MODEL_ONNX_NAME` and `get_deployment_path` wont break anything in v1 or the v2 kv cache pathway?

Reply:
What do you mean exactly? All the tests pass, so the changes should not influence the existing code. What I mean by that is that we only add functionality, not change any existing functionality.